src/Confluent.Kafka/Consumer.cs

```diff
@@ -184,6 +186,11 @@ public void Stop()
             remove { consumer.OnPartitionEOF -= value; }
         }
 
+        public event EventHandler<AggregateException> OnBackgroundPollException;
```

Comment:
I'm currently thinking that background poll functionality would likely be better taken outside the `Consumer` class. Having different ways of operating a consumer (Start/Stop vs Poll or Consume) is not quite as straightforward an interface as I'd like. Now having  `OnBackgroundPollException` as well which is only relevant in the Start/Stop scenario, the situation worse. This is not really the end of the world, but I expect there is a better factoring.